src/components/__tests__/LinkField.test.js

```diff
@@ -0,0 +1,25 @@
+import '@testing-library/jest-dom';
+import { cleanup, screen } from '@testing-library/react';
+import render from '../../test-utils';
+import LinkField from '../LinkField';
+
+afterEach(cleanup);
```

Comment:
iirc this happens automatically now, i.e. you don't need to explicitly have this line yourself